<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_crawling">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>470</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CRAWLING</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="naver">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>701</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>NAVER</string>
   </property>
   <widget class="QLineEdit" name="pagenum">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>56</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>page</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>미엘</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>다운로드 경로</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="download_naver">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>701</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>KAKAO</string>
   </property>
   <widget class="QLineEdit" name="url">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>URL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>다운로드 경로</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="download_kakao">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="statusBar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>791</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
